src/python/interpret/ext/blackbox/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2019 Microsoft Corporation
+# Distributed under the MIT software license
+
+import logging
+import sys
+
+from interpret.ext.extension_utils import load_class_extensions
+
+module_logger = logging.getLogger(__name__)
+
+BLACKBOX_EXTENSION_KEY = "interpret_ext_blackbox"
+
+
+def _is_valid_blackbox_explainer(proposed_blackbox_explainer):
+    return True
+
+
+# How to get the current module
+# https://stackoverflow.com/questions/1676835
+current_module = sys.modules[__name__]
+
+load_class_extensions(current_module, BLACKBOX_EXTENSION_KEY, _is_valid_blackbox_explainer)
```

Comment:
Worth thinking about moving the validation function to be called separately? Just for simplification

Reply:
I liked having one function call for error handling to be simpler but breaking this up could avoid function passing which people don't always like.
I can look into breaking this up into a few function calls:

to_register = get_proposed_class_extensions
valid_classes = [class for class in to_register if _is_valid]
register_class_extensions(valid_classes)